src/azure-cli-testsdk/azure/cli/testsdk/base.py

```diff
@@ -29,27 +27,17 @@
 from .utilities import create_random_name
 from .decorators import live_only
 
-logger = logging.getLogger('azuer.cli.testsdk')
+logger = logging.getLogger('azure.cli.testsdk')
 
 
 class IntegrationTestBase(unittest.TestCase):
     def __init__(self, method_name):
         super(IntegrationTestBase, self).__init__(method_name)
         self.diagnose = os.environ.get(ENV_TEST_DIAGNOSE, None) == 'True'
 
-    def cmd(self, command, checks=None, expect_failure=False):
-        if self.diagnose:
```

Comment:
Is this being removed? I use this extensively during local troubleshooting.  Or is this now accomplished another way?